pombola/south_africa/data/constituencies_and_offices/source_data/2019/eff/eff-2019-data.json

```diff
@@ -0,0 +1,256 @@
+      "Physical Address": "18 Murray Street, Stutterheim, Eastern Cape.", 
+      "Type": "office", 
+      "Source Note": "Economic Freedom Fighters Parliamentary Caucus 2019", 
+      "Postal Address": "18 Murray Street, Stutterheim, Eastern Cape."
+    }, 
+    {
+      "Province": "Gauteng", 
+      "Source URL": "https://www.pivotaltracker.com/file_attachments/100155939/download?inline=true", 
+      "Title": "EFF Constituency Office: Braamfontein", 
+      "Party": "EFF", 
+      "Physical Address": "78 Corner De Korte & De Beer Streets, Mineralia Bld, 3rd Floor, Braamfontein, Gauteng.", 
+      "Type": "office", 
+      "Source Note": "Economic Freedom Fighters Parliamentary Caucus 2019", 
+      "Postal Address": "78 Corner De Korte & De Beer Streets, Mineralia Bld, 3rd Floor, Braamfontein, Gauteng."
+    }, 
+    {
+      "Province": "KwaZulu Natal", 
+      "Source URL": "https://www.pivotaltracker.com/file_attachments/100155939/download?inline=true", 
+      "Title": "EFF Constituency Office: Newcastle", 
+      "Party": "EFF", 
+      "Physical Address": "61 Allen Str, Newcastle, KwaZulu Natal.", 
+      "Type": "office", 
+      "Source Note": "Economic Freedom Fighters Parliamentary Caucus 2019", 
+      "Postal Address": "61 Allen Str, Newcastle, KwaZulu Natal."
+    }, 
+    {
+      "Province": "Free State", 
+      "Source URL": "https://www.pivotaltracker.com/file_attachments/100155939/download?inline=true", 
+      "Title": "EFF Constituency Office: Phuthaditjhaba", 
+      "Party": "EFF", 
+      "Physical Address": "Office 130-131, Setsing 1, Mamponi Str, Phuthaditjhaba, Free State.", 
+      "Type": "office", 
+      "Source Note": "Economic Freedom Fighters Parliamentary Caucus 2019", 
+      "Postal Address": "Office 130-131, Setsing 1, Mamponi Str, Phuthaditjhaba, Free State."
+    },
+    {
+      "Province": "North West",
+      "Source URL": "https://www.pivotaltracker.com/file_attachments/100155939/download?inline=true",
+      "Title": "EFF Constituency Office: Klerksdorp",
+      "Party": "EFF",
+      "Physical Address": "Office 207, Leasik Str, Klerksdorp, North West.",
+      "Type": "office",
+      "Source Note": "Economic Freedom Fighters Parliamentary Caucus 2019",
+      "Postal Address": "Office 207, Leasik Str, Klerksdorp, North West."
+    },
+    {
+      "Province": "Western Cape",
+      "Source URL": "https://www.pivotaltracker.com/file_attachments/100155939/download?inline=true",
+      "Title": "EFF Constituency Office: Cape Town",
+      "Party": "EFF",
+      "Physical Address": "Office 603, 47 Strand Str, Strand Bld, Cape Town, Western Cape.",
+      "Type": "office",
+      "Source Note": "Economic Freedom Fighters Parliamentary Caucus 2019",
+      "Postal Address": "Office 603, 47 Strand Str, Strand Bld, Cape Town, Western Cape."
+    },
+    {
+      "Province": "Free State",
+      "Source URL": "https://www.pivotaltracker.com/file_attachments/100155939/download?inline=true",
```

Comment:
will this be the URL used in the popolo data? That'll be a bit weird and we should perhaps find a more "official" place to store this, even if there's a bog + files feature on pa.org.za

Reply:
Uploaded it to PA and updated the URL :)